src/Utils/utils.ts

```diff
@@ -454,3 +454,22 @@ export const isPostPartum = (data_of_delivery?: string) => {
 export const isAntenatal = (menstruation_start_date?: string) => {
   return dayjs().diff(menstruation_start_date, "month") <= 9;
 };
+
+/**
+ * A utility method to format an array of string to human readable format.
+ *
+ * @param values Array of strings to be made human readable.
+ * @returns Human readable version of the list of strings
+ */
+export const humanizeStrings = (strings: readonly string[]) => {
+  if (strings.length === 0) {
+    throw "Empty array of strings cannot be humanized. Array must contain one or more elements";
```

Comment:
can we just return an empty string here instead of throwing an error, when this function is used to convert an array that is changeable in runtime, this would be hassle to handle, we might need to use try catch.